docs/tutorial/kymographs.rst

```diff
@@ -75,116 +105,148 @@ There are also several properties available for convenient access to the kymogra
-    widget = kymo.crop_and_calibrate(channel="green", tether_length_kbp=48.502)
-    plt.show()
+    Currently this is a static calibration, meaning it is only valid if the traps do not change position during the time of the kymograph.
+
+    Also, the accuracy of the calibration is dependent on how the kymo is cropped. If you crop the kymo by visually estimating the
+    bead edges, the resulting position should be taken as approximate.
+
+We can also interactively slice, crop, and calibrate kymographs using :meth:`~lumicks.pylake.kymo.Kymo.crop_and_calibrate`::
+
+    widget = kymo.crop_and_calibrate(channel="rgb", tether_length_kbp=48.502, aspect="auto", adjustment=lk.ColorAdjustment(0, 99.5, mode="percentile"))
 
 .. image:: figures/kymographs/kymo_interactive.png
 
 Simply click and drag the rectangle selector to the desired ROI. After closing the widget, we can access the edited kymograph
 with::
 
     new_kymo = widget.kymo
+
+    plt.figure()
     new_kymo.plot("green")
+    plt.show()
 
 .. image:: figures/kymographs/kymo_interactive_result.png
 
 If the optional `tether_length_kbp` argument is supplied, the kymograph is automatically calibrated to the desired
 length in kilobase pairs. If this argument is missing (the default value `None`) the edited kymograph is only
 sliced and cropped.
 
-Note that you can also flip a kymograph along its positional axis using :meth:`kymo.flip()
-<lumicks.pylake.kymo.Kymo.flip()>`. This returns a new (but flipped) :class:`~lumicks.pylake.kymo.Kymo`.
-
 Downsampling
 ------------
 
 We can downsample a kymograph in time by invoking::
 
-    kymo_ds = kymo_cropped.downsampled_by(time_factor=2)
-
-.. image:: figures/kymographs/kymo_downsampled_time.png
+    kymo_ds_time = kymo_cropped.downsampled_by(time_factor=2)
 
 Or in space by invoking::
 
-    kymo_ds = kymo_cropped.downsampled_by(position_factor=2)
-
-.. image:: figures/kymographs/kymo_downsampled_position.png
+    kymo_ds_position = kymo_cropped.downsampled_by(position_factor=2)
 
 Or both::
 
     kymo_ds = kymo_cropped.downsampled_by(time_factor=2, position_factor=2)
+    adjustment = lk.ColorAdjustment(0, 30, mode="absolute")
+
+    plt.figure()
+
+    plt.subplot(221)
+    kymo_cropped.plot("green", adjustment=adjustment)
+    plt.title("original")
+    plt.subplot(222)
+    kymo_ds_time.plot("green", adjustment=adjustment)
+    plt.title("downsampled time")
+    plt.subplot(223)
+    kymo_ds_position.plot("green", adjustment=adjustment)
+    plt.title("downsampled position")
+    plt.subplot(224)
+    kymo_ds.plot("green", adjustment=adjustment)
+    plt.title("downsampled both")
+
+    plt.tight_layout()
+    plt.show()
 
-.. image:: figures/kymographs/kymo_downsampled_time_and_position.png
+.. image:: figures/kymographs/kymo_downsampled.png
 
 Note however, that not all functionalities are present anymore when downsampling a kymograph. For
 example, if we downsample a kymograph by time, we can no longer access the per pixel timestamps::
```

Comment:
Maybe mention why?
It's also good to know that downsampling by space still allows more downstream analysis than downsampling by time does.